pkg/logs/input/docker/launcher.go

```diff
@@ -30,6 +32,7 @@ type Launcher struct {
 	cli               *client.Client
 	registry          auditor.Registry
 	stop              chan struct{}
+	lostSocket        chan string
```

Comment:
Same here, the . name of the channel is misleading and does not really reflect what it contains.